buildpipeline/Core-Setup-Windows-arm64.json

```diff
@@ -31,7 +31,7 @@
       },
       "inputs": {
         "filename": "build.cmd",
-        "arguments": "-Configuration Release -Targets Init,Compile,Package,Publish -Architecure x64 -TargetArch arm64 -ToolsetDir c:\\tools\\clr -Framework netcoreapp2.0",
+        "arguments": "-Configuration Release -Targets Init,Compile,Package,Publish -Architecure x64 -TargetArch arm64 -ToolsetDir c:\\tools\\clr -Framework netcoreapp2.0 $(PB_PortableBuild)",
```

Comment:
That's my understanding too. The build steps look like it, and only having "-Signing-" in the names of the x64/x86 defs is some extra evidence (although weak). /cc @crummel